python/ray/tests/test_actor.py

```diff
@@ -925,8 +921,10 @@ def __init__(self):
         def ping(self):
             return
 
+    start = time.time()
     actors = [Actor.remote() for _ in range(num_nodes)]
     ray.get([actor.ping.remote() for actor in actors])
+    assert time.time() - start < 10
```

Comment:
What is the purpose of the timing measurement in this test? We can probably just remove it, right?

Reply:
Hmm true, I forgot that the point of this test was that it used to hang.

Maybe we should just increase the timeout instead.

Reply:
In general, I think it's acceptable if a bug in a PR causes a test to hang (then we'll notice and fix the bug before it gets merged), so I'd be ok with no timeout, but if either of you feel strongly we can keep the timeout and make it longer.

Reply:
ok. Fixed per your comment.